pom.xml

```diff
@@ -716,6 +716,25 @@
         <artifactId>java-driver-shaded-guava</artifactId>
         <version>25.1-jre</version>
       </dependency>
+      <dependency>
```

Comment:
Note: this is in `<dependencyManagement>` section so just defines global version to use, does not add actual dependency.